src/search/pdbs/pattern_database.cc

```diff
@@ -239,6 +248,21 @@ void PatternDatabase::create_pdb(
         }
     }
 
+    if (compute_plan) {
+        /*
+          If computing a plan during Dijkstra, we store, for each state,
+          an operator leading from that state to another state on a
+          strongly optimal plan of the PDB. We store the first operator
+          encountered during Dijstra and only update it if the goal distance
+          of the state was updated or, in the presence of zero-cost operators,
+          the state could be reached with the same cost but with fewer
+          zero-cost operators. For the latter, we need to count the number of
+          zero-cost operators leading to each state.
+         */
```

Comment:
Is this comment still correct? The "strongly optimal"/"fewer zero-cost operator" parts are no longer strictly needed now that we don't break ties uniformly any more. Are they still there?

Reply:
We missed that detail in the discussion with Thomas, too: if we don't take care to consider an operator as the "best" only if it has minimal applications of zero-cost operators before, we are back to the first version where we tested this and where we ran into cycles of zero-cost operators when attempting to extract the plan. So I think this is indeed needed. If not, we can discuss in the sprint meeting how we can avoid doing this.

Reply:
The current version that doesn't update the parent pointers cannot run in cycles. The cycles can only occur if the parent of a node can be updated after it has been expanded.